### DISCUSSION TITLE
用Taro Vue开发H5 APP如何录制视频并上传

### DISCUSSION THREAD

**Discussion by liyouxia920** [NervJS/taro#9854] [Q & A]
看到Taro有自己录制视频的方法Taro.chooseVideo但不支持H5，我是考虑通过Taro Vue在H5端上开发最后通过HBuilderX打包成APP，那这种情况调用chooseVideo有效吗？谢谢！

**Comment by honlyHuang** [comment]
这个暂时不支持 H5 咯，见文档 https://docs.taro.zone/docs/apis/media/video/chooseVideo
